app/src/gui/fields/qrcode/QRCodeFormField.tsx

```diff
@@ -170,13 +170,14 @@ export function QRCodeButton(props: QRCodeButtonProps): JSX.Element {
     let scanCount = 0;
     let scanResult = '';
     const listener = await BarcodeScanner.addListener(
-      'barcodeScanned',
+      'barcodesScanned',
       async result => {
         // require ten consecutive scans of the same code before
         // accepting the scan
```

Comment:
10 consecutive scans of same code? Is this standard behaviour?

Reply:
Just my solution to getting a reliable scan.  Otherwise it returns too quickly.